api/core/migration_helpers.py

```diff
@@ -7,6 +7,14 @@
 
 
 class PostgresOnlyRunSQL(migrations.RunSQL):
+    def __init__(self, *args, sql_file_path: str = None, **kwargs):
+        if sql_file_path:
+            with open(sql_file_path) as f:
+                sql = f.read()
+                super().__init__(sql, *args, **kwargs)
```

Comment:
I think this might do something odd if the user passes `sql` as an argument as well (I _think_ it would put it in the reverse_sql?). We should ensure that only one of `sql` or `sql_file_path` is passed? Or maybe just create a classmethod? Something like: 

```python
@classmethod
def from_sql_file(cls, file_path: str, reverse_sql: str) -> "PostgresOnlyRunSQL":
    with open(file_path) as f:
        return cls(f.read(), reverse_sql=reverse_sql)
```